src/main/resources/root-acl.rdf

```diff
@@ -0,0 +1,8 @@
+@prefix acl: <http://www.w3.org/ns/auth/acl#> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix webac: <http://fedora.info/definitions/v4/webac#> .
+
+<> a acl:Authorization ;
```

Comment:
This may have been part of my confusion about default-ACLs vs. default-Authorizations. This file should probably be named to reflect that it is actually an `Authorization`... maybe `root-authorization.rdf`?